src/js/components/awardv2/shared/AwardPageWrapper.jsx

```diff
@@ -15,7 +14,7 @@ const AwardPageWrapper = ({
     <div className={`award award-${awardType}`}>
         <div className="award__heading">
             <div className="award__info">
-                <div className="award__heading-text">{startCase(awardTypeDescription)}</div>
+                <div className="award__heading-text">{awardTypeDescription.toUpperCase()}</div>
```

Comment:
This isnt gonna work because in some cases we're actually using Title Case. IE for IDVs `/award_v2/68840500`